### PR TITLE
feat: user/forking docs draft

### PR DESCRIPTION
**Pull request by sidharthWeb3auth**
Draft of documents to use and fork this repo

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: docs/home.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+
+
+## _Scan & Pay_
+
+<img src="screenshots/solPay.png" width="700" height="auto" />
+<img src="screenshots/wallet-home-qrScanner.png" width="700" height="auto" />
+
+
+Scan & Pay button on top lets the user harness [Solana Pay](https://solanapay.com/), which is an decentralized and performant payment network for immediate transactions in SOL or SPL tokens like USDC.
```
Comment (shahbaz17) [inline docs/home.md:48]: ~an~ -> a decentralized

Comment (sidharthWeb3auth) [reply docs/home.md]: @shahbaz17 fixed

File: docs/howToFork.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- 1. Get your Client ID from Web3Auth [Dashboard](https://dashboard.web3auth.io/) by creating a new project ([ How to generate a clientId from Web3Auth dashboard ](howToClientId.md)). [Readmore](https://web3auth.io/docs/quick-start)
+
+- 2. In config.ts file, replace the clientId with your Client ID
+
+- 3. Generate your own sentry and google analytics credentials and use them in .env files.
+
+- 4. API calls to https://solana-api.tor.us are restricted. Please contact our team for help.
+
+- 5. Generate a different sentry dsn for your fork. If use the one available in the current env, all the errors logs will be lost.
```
Comment (shahbaz17) [inline docs/howToFork.md:17]: @sidharthWeb3auth 

Can you also look at this line "If use the one available in the current env," ?

Comment (sidharthWeb3auth) [reply docs/howToFork.md]: @shahbaz17 : done, updated.

**Review by shahbaz17 — APPROVED** [review]
LGTM @sidharthWeb3auth

**Pull request merged**